patterns/footer-newsletter.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Title: Footer with newsletter signup
+ * Slug: twentytwentyfive/footer-newsletter
+ * Categories: footer
+ * Block Types: core/template-part/footer
+ * Description: Footer with large site title and newsletter signup
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--60);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)">
+<!-- wp:site-title {"level":0,"style":{"typography":{"fontSize":"10vw"}}} /-->
+
+	<!-- wp:group {"style":{"spacing":{"padding":{"right":"0","left":"0"}}},"layout":{"type":"constrained","justifyContent":"left"}} -->
```

Comment:
This while section should be a separate pattern, which still needs to be created.